src/cable/channel.cr

```diff
@@ -48,6 +48,11 @@ module Cable
       Cable.server.publish("#{channel}", message.to_json)
     end
 
+    def self.broadcast_to(channel : String, message : String)
+      Cable::Logger.info "[ActionCable] Broadcasting to #{channel}: #{message}"
+      Cable.server.publish("#{channel}", message.to_json)
```

Comment:
I wonder if `to_json` should be called here 🤔  If you pass in a json string, it'll double escape the json meaning you'd have to JSON.parse twice on the other side...

For example:

```crystal
# this could get passed in
json = ["a", "b"].to_json
# json == %(["a","b"])
ChatChannel.broadcast_to("chat", json)

# then on the other side you'd have to do this
# because the data had `to_json` called twice on it
# message == %("[\"a\",\"b\"]")
JSON.parse(JSON.parse(message))
```

What are your thoughts?

Reply:
@jwoertink it works fine as the server is parsing before "encoding" again.
But sure, it seems to work without to_json as well so I can remove that.

Reply:
@jwoertink I changed it back as it made the tests fail. Seems to work better with to_json with the current code.